kbase_cache_client/kbase_cache_client.py

```diff
@@ -3,46 +3,38 @@
 import configparser
 import os
 from pprint import pprint as pp
-from .exceptions import NoCacheIdentifiers, HTTPRequestError, UnknownRequestError, DownloadDirNotWriteable, \
-    CacheNonexistent, AuthorizationTokenNotSet
+from .exceptions import (NoCacheIdentifiers, HTTPRequestError,
+                         UnknownRequestError, DownloadDirNotWriteable,
+                         CacheNonexistent, AuthorizationTokenNotSet)
 
 config = configparser.ConfigParser()
 if os.path.exists('test.cfg'):
     config.read('test.cfg')
-
     if not config.get('KBASE_CACHE_SERVICE', 'TOKEN', fallback=None):
         if not os.getenv('KBASE_CACHE_TOKEN', None):
             raise IOError('Please set your service token in test.cfg, as:\n'
                           '[KBASE_CACHE_SERVICE]\nTOKEN=<token>.\n'
                           'Or as an environmental variable KBASE_CACHE_TOKEN\n'
                           'Consult KBase Administrators if you are not sure how to generate a token')
-else:
-    if not os.getenv('KBASE_CACHE_TOKEN', None):
-        raise IOError('Please create test.cfg and set your service token as:\n'
-                      '[KBASE_CACHE_SERVICE]\nTOKEN=<token>\n'
-                      'Or as an environmental variable KBASE_CACHE_TOKEN\n'
-                      'Consult KBase Administrators if you are not sure how to generate a token')
 
 
 class KBaseCacheClient:
     def generate_cacheid(self, identifiers):
         if not isinstance(identifiers, dict):
             raise NoCacheIdentifiers('Identifiers for cache id must be in dictionary format.')
-
         headers = {'Content-type': 'application/json', 'Authorization': self.service_token}
-
         if not self.callback.endswith('/'):
             endpoint = self.callback + '/cache/v1/cache_id'
         else:
             endpoint = self.callback + 'cache/v1/cache_id'
-
-        req_call = requests.post(endpoint, data=json.dumps(identifiers), headers=headers).json()
-
-        if req_call.get('error'):
-            pp(req_call)
-            raise HTTPRequestError(req_call.json().get('error'))
+        response = requests.post(endpoint, data=json.dumps(identifiers), headers=headers)
```

Comment:
If they enter the wrong service url or auth token, we should catch this and fail early.